pkg/cmd/pr/status/status.go

```diff
@@ -230,7 +230,15 @@ func printPrs(w io.Writer, totalCount int, prs ...api.PullRequest) {
 			} else if reviews.ReviewRequired {
 				fmt.Fprint(w, utils.Yellow("- Review required"))
 			} else if reviews.Approved {
-				fmt.Fprint(w, utils.Green("✓ Approved"))
+				var totalApprovals int
+
+				for _, review := range pr.Reviews.Nodes {
+					if review.State == "APPROVED" {
+						totalApprovals++
```

Comment:
Note: this will count repeated approvals by the same person as multiple approvals.

I sometimes approve a single PR multiple times, usually to acknowledge that I've scanned over code updates since I've last reviewed. Multiple approvals from the same person should technically count as a single approval.

Moreover, if I leave 2 reviews where the first one is Approved and the second one is Request Changes, then my previous review should be discarded and should not qualify as an approval. Only the (chronologically) latest review status (other than a Comment status, which is neutral) from a single person should ever be taken into account.

See https://github.com/cli/cli/blob/c77e6afb0384e778693a3f9d161dffb144dd50a3/pkg/cmd/pr/view/view.go#L257